Project/Src/Gui/TextAreaControl.cs

```diff
@@ -171,28 +176,12 @@ public void UpdateLayout()
             var view = TextArea.TextView;
 
             var currentVisibilites = GetScrollVisibilities(HScrollBar.Visible, VScrollBar.Visible);
-            var visited = new HashSet<ScrollVisibilities>();
+            var bounds = Measure(currentVisibilites);
+            var newVisibilities = ComputeScrollBarVisibilities(bounds.textArea.Size);
+            bounds = Measure(newVisibilities.visibilities);
+            ApplyLayout(newVisibilities.visibilities, newVisibilities.maxLength, bounds);
```

Comment:
We use to separate the function code from nested functions by means of a `return` statement.
```suggestion
            ApplyLayout(newVisibilities.visibilities, newVisibilities.maxLength, bounds);
            
            return;
```